xc/device.ml

```diff
@@ -795,12 +792,14 @@ module NetSriovVf = struct
 
   let add  ~xs ~devid ~mac ?mtu ?(rate=None) ?(backend_domid=0) ?(other_config=[]) 
       ~pci ~vlan ~carrier ?(extra_xenserver_keys=[]) (task: Xenops_task.task_handle) domid =
-    let open Xenops_interface.Pci in
+    let dbg = Printf.sprintf "Task %s reference %s"
```

Comment:
Just `let dbg = Xenops_task.get_dbg task in` should do.

Reply:
Fixed. Thanks.